elements/rh-table/rh-table.ts

```diff
@@ -1,11 +1,16 @@
 import { LitElement, html } from 'lit';
 import { customElement } from 'lit/decorators/custom-element.js';
+import { property } from 'lit/decorators/property.js';
+import { classMap } from 'lit/directives/class-map.js';
 
 import { Logger } from '@patternfly/pfe-core/controllers/logger.js';
 
-import styles from './rh-table.css';
 import { RequestSortEvent, RhSortButton } from './rh-sort-button.js';
 
+import { colorContextConsumer, type ColorTheme } from '../../lib/context/color/consumer.js';
```

Comment:
demo and lightdom css (and my suggestion for editing the changelog) have color palettes as well as themes on table, but i'm only seeing theme here

Reply:
Are you suggesting making table itself a provider too add direct support of `color-palette` attribute? 

@marionnegp and @coreyvickery and I had a discussion where we thought it would be better if table wasn't itself a provider, but just a consumer.  There would need to be a parent element that was providing that context.    Can you elaborate on why you think it should be a provider as well?

Reply:
after poking around a bit more I think I understand the comment now.  the lightdom css actually wouldn't have worked as intended given a nested structure.  Only way to handle that is to have the component react with something styleable  on the host, hence the need for the color palette.